proto/resource_usage_agent.proto

```diff
@@ -31,4 +35,14 @@ message CPUTimeRecord {
 
 }
 
+message SummaryRecord {
+    bytes resource_group_tag = 1;
+
+    // The number of reads of keys associated with resource_group_tag.
+    uint32 read_keys = 2;
+
+    // The number of writes of keys associated with resource_group_tag.
+    uint32 write_keys = 3;
```

Comment:
Do you plan to distinguish index keys and row keys?

Reply:
Can we use the label(used to distinguish row and index) as part of tipb.ResourceGroupTag? In this case, tikv does not need to perceive any difference between `row` and `index`, nor does it need to add additional fields. It only needs to transparently pass the ResourceGroupTag to the agent, and the agent will make a semantic distinction.

Reply:
Got it. This is a brilliant idea!